pages/app-layout/with-custom-triggers.page.tsx

```diff
@@ -0,0 +1,152 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React, { useState } from 'react';
+import { AppLayout, Header, HelpPanel, Link, SegmentedControl, SpaceBetween, SplitPanel } from '~components';
+import appLayoutLabels from './utils/labels';
+import { Breadcrumbs, Containers } from './utils/content-blocks';
+import ScreenshotArea from '../utils/screenshot-area';
+
+export default function () {
+  const [isToolsOpen, setIsToolsOpen] = useState(false);
+  const [toolsContent, setToolsContent] = useState('info');
+  const [triggerStatus, setTriggerStatus] = useState('custom-triggers');
+
+  const customTriggers =
+    triggerStatus !== 'custom-triggers'
+      ? null
+      : {
+          toolsTriggers: [
+            {
+              ariaExpanded: isToolsOpen,
+              ariaLabel: 'View Security content',
+              iconName: 'security',
+
+              onClick: () => {
```

Comment:
fyi I have a big refactoring of focus management in progress, these tools triggers will need 1) consideration of what the focus management behavior should be and 2) probably rebasing once the refactoring is merged https://github.com/cloudscape-design/components/pull/892